.circleci/config.yml

```diff
@@ -17,14 +17,14 @@ executors:
 
   machine_integration_test_exec:
     machine: # run the steps with Ubuntu VM
-      image: ubuntu-2204:2023.10.1
+      image: ubuntu-2204:2024.04.4
```

Comment:
Why is the tag different for this and the `toil_wes_test_executor` below?

Reply:
A sad reason
While experimenting, I bumped to a newer version to try to get a newer Java version on the machine executor. 
It turned out to be unnecessary, but since we ran the tests anyway I kept the newer versions that worked. 

Something about `ubuntu-2204:2024.11.1` breaks the toil install silently.